apps/api/v2/src/modules/organizations/organizations.repository.ts

```diff
@@ -16,6 +16,7 @@ export class OrganizationsRepository {
     return this.dbRead.prisma.team.findUnique({
       where: {
         id: organizationId,
+        isOrganization: true,
```

Comment:
@joeauyeung is relying on "isOrganization: true" the same as checking if parentId is null of an org? By default "isOrganization" is false, so I am wondering if data was not manually updated in prod whether or not its safer to check if parentId is null rather than if "isOrganization" is true.

Reply:
I think `parentId=null` would allow this to select nonOrg teams. A normal team in the DB has no parentId either 🤔 

All Orgs should have `isOrganization=true` and also `organizationSettings` if you want to go that route